prac_06/guitar.py

```diff
@@ -0,0 +1,30 @@
+"""
+Word Occurrences
+Estimate: 30 minutes
+Actual:   50 minutes
+"""
+class Guitar:
+    """Represent a Guitar object."""
+    def __init__(self, name="",year = 0 , cost = 0 ):
+        self.name = name
+        self.year = year
+        self.cost = cost
+
+    def __str__(self):
+        """Initialise a guitar instance."""
+        return f"{self.name} get_age() - Expected {self.get_age()}. Got {self.get_age()}\n{self.name} is_vintage() - Expected {self.is_vintage()}. Got {self.is_vintage()}"
+        # return f"{self.name} ,{self.year} : {self.cost}"
+
+
+    def get_age(self):
+        """ Calculate the age of guitar"""
+        current_year = 2024
+        return current_year-self.year
```

Comment:
If you are adding fixed number, that will not change. It is better to use as a CONSTANT